packages/sit-onyx/src/components/OnyxRadioButtonGroup/OnyxRadioButtonGroup.vue

```diff
@@ -12,7 +13,10 @@ const props = defineProps<{
    */
   name: string;
   modelValue?: SelectionOption<T>;
-  label?: string;
+  /**
+   * Headline and legend shown above the radio button group
```

Comment:
Its only one headline, right? Not both a headline and legend
```suggestion
   * Headline to show above the radio button group
```

Reply:
Yes, I clarified it. Fine like that? :)